muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/credentials/body/credentials-hero.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import mApi from '~/lib/mApi';
+import promisify from '~/util/promisify';
+import "~/sass/elements/credentials.scss";
+import {i18nType} from '~/reducers/base/i18n';
+
+interface CredentialsHeroProps {
+  modifier? : string,
+  i18n: i18nType
+}
+
+interface CredentialsHeroState {
+  hash: string
+}
+
```

Comment:
There are multiple empty lines that should be removed to preserve code's best readability.